packages/common/env.ts

```diff
@@ -23,9 +23,11 @@ export const env = createEnv({
       .regex(/^[0-9a-fA-F]{64}$/, {
         message: `SECRET_ENCRYPTION_KEY must only contain hex characters${errorSuffix}`,
       }),
+    UNSAFE_NO_EXTERNAL_CONNECTION: createBooleanSchema(false),
```

Comment:
Is it really "usafe"? IMO having no internet can be certainly a use case which we should plan for

Reply:
Okay sure, I'll rename it. I just though that it is not a variable normal users (that do not have this usecase) should use